File: app/src/main/java/com/shevelev/comics_viewer/activities/folders/ChooseFoldersOptionsMenuManager.java
package com.shevelev.comics_viewer.activities.folders;

import android.view.Menu;

import com.shevelev.comics_viewer.common.OptionsMenuManager;

public class ChooseFoldersOptionsMenuManager extends OptionsMenuManager
{
    public ChooseFoldersOptionsMenuManager(Menu menu)
    {
        super(menu);
    }

    public void setAcceptVisible(boolean visible)
    {
        setVisible(1, visible);
    }
}

File: app/src/main/java/com/shevelev/comics_viewer/ui/activities/folders/ChooseFoldersOptionsMenuManager.kt
package com.shevelev.comics_viewer.ui.activities.folders

import android.view.Menu
import com.shevelev.comics_viewer.common.OptionsMenuManager

class ChooseFoldersOptionsMenuManager(menu: Menu) : OptionsMenuManager(menu) {
    fun setAcceptVisible(visible: Boolean) {
        setVisible(1, visible)
    }
}